src/main/resources/static/cas2-api.yml

```diff
@@ -62,6 +67,15 @@ paths:
                 type: array
                 items:
                   $ref: '_shared.yml#/components/schemas/ApplicationSummary'
+          headers:
+            X-Pagination-CurrentPage:
+              $ref: '#/components/headers/X-Pagination-CurrentPage'
```

Comment:
Does this work for you if you try and view the OpenAPI spec in IntelliJ (top right of your editor, see screenshot)? My branch highlighted the other occurrence of this as an error so I had to make this `_shared.yml#/components/headers/X-Pagination-CurrentPage` - not sure if it's just a me thing or if you're seeing any issues?
<img width="279" alt="image" src="https://github.com/ministryofjustice/hmpps-approved-premises-api/assets/19826940/4de2b508-d5f0-4322-8249-a0c512dc434b">

Reply:
I am yes.  Missed this earlier.  I've added a more recent commit to address this issue in 8 places (4 for /applications and 4 for /submissions).  Would you mind re-approving George then I will release this morning?

@Harriethw tagged for info